lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -90,10 +96,19 @@ def edit_subscriber(self, subscriber_id):
             subscriber_data.ParseFromString(row[0])
             yield subscriber_data
             data_str = subscriber_data.SerializeToString()
-            self.conn.execute(
-                "UPDATE subscriberdb SET data = ? " "WHERE subscriber_id = ?",
-                (data_str, subscriber_id),
-            )
+            try:
+                self.conn.execute(
```

Comment:
Can you use this as a context manager https://docs.python.org/2/library/sqlite3.html#using-the-connection-as-a-context-manager

Reply:
it already is.